scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1214,30 +1218,40 @@ class Router(formatOps: FormatOps) {
         )
       // non-statement starting curly brace
       case FormatToken(_: T.Comma, open: T.LeftBrace, _)
-          if !style.poorMansTrailingCommasInConfigStyle && {
-            if (isCallSite(leftOwner)) !style.binPack.unsafeCallSite
-            else isDefnSite(leftOwner) && !style.binPack.unsafeDefnSite
-          } =>
+          if !style.poorMansTrailingCommasInConfigStyle &&
```

Comment:
So `isDefnSite` is no longer handled in this case and should be handled above in 783, right?

Reply:
actually, the thought was that defn site looks impossible for this rule. comma is followed by a parameter name which can't start with a brace... can it?

Reply:
Ach yeah! Makes sense, I can't find any scenario that this would fit in.